src/styles/Book.module.css

```diff
@@ -0,0 +1,68 @@
+.displayFlex {
+  display: flex;
+}
+
+.alignCenter {
+  align-items: center;
+}
+
+.bookContainer {
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 32px 27px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  border: 1px solid #e8e8e8;
+}
+
+.bookCategory {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 600;
+  color: #8b8b8b;
+}
+
+.bookTitle {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 600;
+  font-size: 22px;
+}
+
+.bookLinks {
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+  color: #4386bf;
+  font-weight: 100;
+  margin-right: 10px;
+}
+
+.removeBook {
+  background-color: #fff;
+  border: none;
+}
+
+.completionPercentage {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 32px;
+}
+
+.completed {
+  color: #8b8b8b;
+  font-size: 15px;
+}
+
+.chapter {
+  font-family: 'Roboto', sans-serif;
+}
+
+.chapterTitle {
+  color: #8b8b8b;
+}
+
+.updateProgress {
+  background-color: #4386bf;
+  color: #fff;
+  border: none;
+  padding: 8px 18px;
+  border-radius: 4px;
+}
```

Comment:
To avoid repeatition of css rules, rewrite some of the instructions in this module like so:

```
 .bookTitle, .bookLinks, .chapter {
  font-family: 'Roboto', sans-serif;
}

.bookCategory, .completionPercentage {
  font-family: 'Montserrat', sans-serif;
}
```